diff --git a/src/ap/wpa_auth.c b/src/ap/wpa_auth.c
index 070236a89..e49d60c20 100644
--- a/src/ap/wpa_auth.c
+++ b/src/ap/wpa_auth.c
@@ -2172,11 +2172,12 @@ SM_STATE(WPA_PTK, PTKSTART)
 	 * cases would also apply here, but at least in the IBSS case, this
 	 * would cover a potential real use case.
 	 */
-	if (sm->wpa == WPA_VERSION_WPA2 &&
-	    (wpa_key_mgmt_wpa_ieee8021x(sm->wpa_key_mgmt) ||
-	     (sm->wpa_key_mgmt == WPA_KEY_MGMT_OWE && sm->pmksa) ||
-	     wpa_key_mgmt_sae(sm->wpa_key_mgmt)) &&
-	    sm->wpa_key_mgmt != WPA_KEY_MGMT_OSEN) {
+	// if (sm->wpa == WPA_VERSION_WPA2 &&
+	//     (wpa_key_mgmt_wpa_ieee8021x(sm->wpa_key_mgmt) ||
+	//      (sm->wpa_key_mgmt == WPA_KEY_MGMT_OWE && sm->pmksa) ||
+	//      wpa_key_mgmt_sae(sm->wpa_key_mgmt)) &&
+	//     sm->wpa_key_mgmt != WPA_KEY_MGMT_OSEN) {
+	if (sm->wpa == WPA_VERSION_WPA2) {	
 		pmkid = buf;
 		pmkid_len = 2 + RSN_SELECTOR_LEN + PMKID_LEN;
 		pmkid[0] = WLAN_EID_VENDOR_SPECIFIC;
